frontends/election-manager/src/app.test.tsx

```diff
@@ -139,29 +138,30 @@ async function authenticateWithAdminCard(card: MemoryCard) {
   card.insertCard(
     makeAdminCard(eitherNeitherElectionDefinition.electionHash, '123456')
   );
-  jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
   await screen.findByText('Enter the card security code to unlock.');
-  fireEvent.click(screen.getByText('1'));
-  fireEvent.click(screen.getByText('2'));
-  fireEvent.click(screen.getByText('3'));
-  fireEvent.click(screen.getByText('4'));
-  fireEvent.click(screen.getByText('5'));
-  fireEvent.click(screen.getByText('6'));
+  userEvent.click(screen.getByText('1'));
+  userEvent.click(screen.getByText('2'));
+  userEvent.click(screen.getByText('3'));
+  userEvent.click(screen.getByText('4'));
+  userEvent.click(screen.getByText('5'));
+  userEvent.click(screen.getByText('6'));
   await screen.findByText('Remove card to continue.');
   card.removeCard();
-  jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
```

Comment:
I've started removing some of these `jest.advanceTimersByTime(CARD_POLLING_INTERVAL)` calls, specifically in UI layer tests where we can just rely on `await screen.findBy` calls